### ISSUE TITLE
bug: DSL to JSON grammar parsing is failing in the browser

### ISSUE THREAD

**Issue by dbrrt** [openfga/syntax-transformer#51]
I'm experiencing some difficulties integrating the syntax transformer in a NextJS application.

Versions:
- `nextJS`: `12.1.6`
- `@openfga/syntax-transformer`: `0.0.4`

I've added this to my `next.config.js` / webpack config as nearley has a dependency to `fs`

```js
 webpack: (config, { isServer }) => {
    if (!isServer) {
      config.resolve.fallback.fs = false;
    }
    return config;
  }
```

Here's my integration in a React component in the nextJS app (works well in a nodeJS app):

```js
      const apiSyntax = friendlySyntaxToApiSyntax(`type document
      relations
        define blocked as self
        define editor as self but not blocked
    type team
      relations
        define member as self
    `);
```

and my stack error trace:

```js
Error: Syntax error at line 2 col 7:

1 type document
2       relations
        ^

Unexpected "r". Instead, I was expecting to see one of the following:

A " " based on:
    _optional_space$ebnf$1 → _optional_space$ebnf$1 ● " "
    _optional_space →  ● _optional_space$ebnf$1
    _newline →  ● _optional_space "\n"
    type$ebnf$2$subexpression$2 →  ● _newline
    type$ebnf$2 → type$ebnf$2 ● type$ebnf$2$subexpression$2
    type → type$ebnf$1 _type _naming ● type$ebnf$2
    types$ebnf$2$subexpression$1 →  ● type relations types$ebnf$2$subexpression$1$ebnf$1
    types$ebnf$2 → types$ebnf$2 ● types$ebnf$2$subexpression$1
    types → types$ebnf$1 ● types$ebnf$2
A " " based on:
    _comment$ebnf$1 → _comment$ebnf$1 ● " "
    _comment →  ● _comment$ebnf$1 "#" _spacing _comment$ebnf$2 _comment$ebnf$3 _newline
    relations$ebnf$1$subexpression$1 →  ● _comment
    relations$ebnf$1 → relations$ebnf$1 ● relations$ebnf$1$subexpression$1
    relations →  ● relations$ebnf$1 _relations relations$ebnf$2
    types$ebnf$2$subexpression$1 → type ● relations types$ebnf$2$subexpression$1$ebnf$1
    types$ebnf$2 → types$ebnf$2 ● types$ebnf$2$subexpression$1
    types → types$ebnf$1 ● types$ebnf$2
A "#" based on:
    _comment → _comment$ebnf$1 ● "#" _spacing _comment$ebnf$2 _comment$ebnf$3 _newline
    relations$ebnf$1$subexpression$1 →  ● _comment
    relations$ebnf$1 → relations$ebnf$1 ● relations$ebnf$1$subexpression$1
    relations →  ● relations$ebnf$1 _relations relations$ebnf$2
    types$ebnf$2$subexpression$1 → type ● relations types$ebnf$2$subexpression$1$ebnf$1
    types$ebnf$2 → types$ebnf$2 ● types$ebnf$2$subexpression$1
    types → types$ebnf$1 ● types$ebnf$2
A "\n" based on:
    _newline → _optional_space ● "\n"
    type$ebnf$2$subexpression$2 →  ● _newline
    type$ebnf$2 → type$ebnf$2 ● type$ebnf$2$subexpression$2
    type → type$ebnf$1 _type _naming ● type$ebnf$2
    types$ebnf$2$subexpression$1 →  ● type relations types$ebnf$2$subexpression$1$ebnf$1
    types$ebnf$2 → types$ebnf$2 ● types$ebnf$2$subexpression$1
    types → types$ebnf$1 ● types$ebnf$2
```

**Comment by rhamzeh** [comment]
Hi @dbrrt - we synced on this offline a while back and you mentioned the issue has been resolved (as of the [v0.0.5](https://github.com/openfga/syntax-transformer/releases/tag/v0.0.5) release).

Just wanted to double back and make sure that's still the case before I move the issue to resolved.

**Comment by rhamzeh** [comment]
Completed as part of #52 released in [v0.0.5](https://github.com/openfga/syntax-transformer/releases/tag/v0.0.5)
